spec/integ/crypto/cross-signing.spec.ts

```diff
@@ -236,6 +237,53 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("cross-signing (%s)", (backend: s
                 `[${TEST_USER_ID}].[${TEST_DEVICE_ID}].signatures.[${TEST_USER_ID}].[ed25519:${SELF_CROSS_SIGNING_PUBLIC_KEY_BASE64}]`,
             );
         });
+
+        it("can bootstrapCrossSigning twice", async () => {
+            mockSetupCrossSigningRequests();
+
+            const authDict = { type: "test" };
+            await bootstrapCrossSigning(authDict);
+
+            // a second call should do nothing except GET requests
```

Comment:
What get requests?

Reply:
It does `GET /user/:user/account_data/:type` requests, because this test is running before the initialsync completes (so it has not cached any account data in memory).

Would you find this useful to include in the comment?

Reply:
No maybe not. Was trying to remember why it would do requests.
Looks like it's checking if it's in 4S or not to eventually update 4S or import. There are some comments about it in bootstrapCrossSigning. I guess the naming is not perfect, maybe it's ensureCrossSigning but also fix it :/